crypto/sha/asm/sha256-loongarch64.pl

```diff
@@ -42,6 +42,9 @@
 my $output = $#ARGV >= 0 && $ARGV[$#ARGV] =~ m|\.\w+$| ? pop : undef;
 my $flavour = $#ARGV >= 0 && $ARGV[0] !~ m|\.| ? shift : undef;
 
+my $use_lsx = $flavour && $flavour =~ /lsx/i ? 1 : 0;
+my $isaext = "_" . ( $use_lsx ? "lsx" : "la64v100" );
```

Comment:
Note that "Toolchain Conventions of the LoongArch(TM) Architecture" depicts -march=la64v1.0 meaning *supporting* LSX, and the latest GCC & Clang follows it.  Thus it's better to change the naming to avoid confusion.